public/app/plugins/panel/dashlist/module.tsx

```diff
@@ -52,9 +52,9 @@ export const plugin = new PanelPlugin<Options>(DashList)
         defaultValue: defaultOptions.query,
       })
       .addCustomEditor({
-        path: 'folderUid',
+        path: 'folderUID',
```

Comment:
assume this is just a naming consistency thing?

Reply:
Yes - folderUID was the name of the property being used.